lib/services/application_services.ex

```diff
@@ -11,6 +11,12 @@ defmodule ApplicationRunner.ApplicationServices do
 
   @empty_body "{}"
   @empty_body_length "2"
+  @min_scale_label "com.openfaas.scale.min"
+  @max_scale_label "com.openfaas.scale.max"
+  @scale_factor_label "com.openfaas.scale.factor"
+  @min_scale_default "0"
+  @max_scale_default "5"
+  @scale_factor_default "10"
```

Comment:
Maybe use config to facilitate maintenance, at least for `min_scale_label` `max_scale_label` they will probably evolve over time over time

Reply:
No they won't evolve since they are OpenFaaS labels.